common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -576,18 +576,19 @@ bool mon_wlan_hal_dwpal::update_vap_stats(const std::string &vap_iface_name, SVa
     }
 
     /* TEMP: Traces... */
-    LOG(DEBUG) << "GET_VAP_MEASUREMENTS reply= \n" << reply;
-    LOG(DEBUG) << "numOfValidArgs[0]= " << numOfValidArgs[0] << " BytesSent= " << BytesSent;
-    LOG(DEBUG) << "numOfValidArgs[1]= " << numOfValidArgs[1] << " BytesReceived= " << BytesReceived;
-    LOG(DEBUG) << "numOfValidArgs[2]= " << numOfValidArgs[2] << " PacketsSent= " << PacketsSent;
-    LOG(DEBUG) << "numOfValidArgs[3]= " << numOfValidArgs[3]
-               << " PacketsReceived= " << PacketsReceived;
-    LOG(DEBUG) << "numOfValidArgs[4]= " << numOfValidArgs[4]
-               << " RetransCount= " << vap_stats.retrans_count;
-    LOG(DEBUG) << "numOfValidArgs[5]= " << numOfValidArgs[5]
-               << " ErrorsSent= " << vap_stats.errors_sent;
-    LOG(DEBUG) << "numOfValidArgs[6]= " << numOfValidArgs[6]
-               << " ErrorsReceived= " << vap_stats.errors_received;
+    //        LOG(DEBUG) << "GET_VAP_MEASUREMENTS reply= \n" << reply;
```

Comment:
(nitpick) Please don't leave code in comments, remove it instead (we can always look at the git history if we want it back).